spec/lib/importers/case_importer_spec.rb

```diff
@@ -18,13 +18,13 @@
       expect { case_importer.import_cases }.to change(CasaCase, :count).by(3)
 
       # correctly imports true/false transition_aged_youth
-      expect(CasaCase.find_by(case_number: "CINA-01-4347").transition_aged_youth).to be_truthy
-      expect(CasaCase.find_by(case_number: "CINA-01-4348").transition_aged_youth).to be_falsey
+      expect(CasaCase.find_by(case_number: "CINA-01-4347").transition_aged_youth).to be_falsey # birth_month_year_youth is nil
```

Comment:
If you don't expect to have this field anymore, you could remove this expect or change it to `be_nil` that corresponds better to the expectation.

Reply:
I generally prefer that we follow https://www.betterspecs.org/ but as long as the linter likes it, I'm ok with it.